### PR TITLE
Update changelog for 5.0.0 release

### PR DESCRIPTION
**Pull request by rahulporuri**
This PR updates the changelog in prep for the upcoming 5.0.0 release. ~Note that there are two PRs that are yet to be merged - which are release blockers for the 5.0.0 release - which have not been included in this changelog.~ Changelog entries for #251 and #571 are now included in this PR. This can now be merged.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,18 +1,149 @@
...
+* Clean up some of the agg tests. (#573)
+* Add a unit test for drawing images. (#572)
+* Add ``tox.ini`` for flake8 control. (#559)
+* Unskip most of the ``TestGLDrawing`` tests on OSX. (#540)
+* Address resource warnings because of open file handles. (#529)
+* Rename test module so it gets discovered by unittest. (#528)
+* Fix tests for PySide2 and add PySide2 to the testing matrix. (#484)
+* Make tests for font manager quieter. (#487)
+* Use unittest as the test runner instead of nose. (#475)
+* Switch on default warnings flag for CI test command. (#469)
```
Comment (aaronayres35) [inline CHANGES.txt:66]: the distinction between some of the items in this section vs Build and CI is a little unclear to me, but its pretty inconsequential.  Not a suggested change

File: CHANGES.txt [hunk 2]
```diff
@@ -1,18 +1,149 @@
...
+--------------------------------
+
+* Update ``setup.py`` to allow creation of release candidates. (#575)
+* Update CI status badge. (#579)
+* Provide the right path to the version file on windows. (#563)
+* Unconditionally list ``pillow`` as a runtime dependency. (#562)
+* Move to a single ``setup.py`` file. (#535)
+* Flush Travis and Appveyor CI. (#549)
+* Add null to the list of toolkits tested on GitHub Actions CI. (#548)
+* Mention in the README ``libglu1-mesa-dev`` is a build dependency on Linux. (#546)
```
Comment (aaronayres35) [inline CHANGES.txt:123]: this could be moved to documentation?

File: CHANGES.txt [hunk 3]
```diff
@@ -1,18 +1,149 @@
...
+* Fix a number of issues with the examples. (#402)
+* Small fixes for examples. (#388)
+* Add a demo runner for enable. (#383)
+
+Testing
+-------
+
+* Clean up some of the agg tests. (#573)
+* Add a unit test for drawing images. (#572)
+* Add ``tox.ini`` for flake8 control. (#559)
```
Comment (aaronayres35) [inline CHANGES.txt:59]: maintenance?

**Review by aaronayres35 — APPROVED** [review]
LGTM!  Quite a lengthy list of changes.  I left some very nit picky comments about shuffling a few changes into other categories, but they are pretty inconsequential so feel free to ignore.  

Also I'm assuming we are following the old enable convention of not including release summary/highlights and thanks to contributors as we do else where in ets.  Consistency within the enable changelog makes sense to me 👍 .  We may eventually want to add these for cross ets consistency though

**Comment by rahulporuri** [comment]
> ... and thanks to contributors as we do else where in ets

I added this section and I addressed your comments. I haven't put together a summary though.

**Comment by rahulporuri** [comment]
@aaronayres35 if you're happy with the changelog in this PR, we can merge it and go ahead with the release candidate today.

**Review by aaronayres35 — APPROVED** [review]
LGTTM

**Pull request merged**